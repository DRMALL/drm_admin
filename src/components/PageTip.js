

import React from 'react'

const PageTip = ({tip}) => (
  <div className='page-tip' >{ tip }</div>
  )

export default PageTip
