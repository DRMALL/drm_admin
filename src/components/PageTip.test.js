

import PageTip from './PageTip'
import React from 'react'
import renderer from 'react-test-renderer'


it('PageTip test', ()=> {
  const tip = '标题示例'
  const tree = renderer.create(<PageTip tip={ tip } />)
  expect(tree).toMatchSnapshot()
})
