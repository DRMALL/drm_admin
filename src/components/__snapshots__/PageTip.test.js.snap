// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTip test 1`] = `
<div
  className="page-tip"
>
  标题示例
</div>
`;
